### DISCUSSION TITLE
Viscidus spell/elemental damage types and CC's intentions

### DISCUSSION THREAD

**Discussion by trinitarius** [chromiecraft/chromiecraft#4209] [General]
CC intends to give a "blizzlike" experience of 3.3.5 WotLK by improving Azerothcore. The following discussion isn't worth to be posted as a bug or issue. During some reading regarding the whole Viscidus frost encounter tactics it was obvious that the role of wands during the initial state of the game for this fight is controversial and not clearly documented, resulting in discussion within the community and Blizzard.

**Confirmed working on retail/vanilla/classic and CC:**
- Frost Oil.
- Any weapon or item with a frost spell damage proc or function (e.g. Coldrage Dagger).
- Frost spells of any class.

**Confirmed not working on retail/vanilla/classic and CC:**
- Weapons with additional frost damage (Iceblade Hacker, Warblade of Caer Darrow)

**Not enough information to confirm working/not working:**
- Frost wands (unclear if they worked during vanilla/tbc, doesn't work in retail, contradicting information for timeline 3.0.9-3.3.5; patch notes empty, only vague mentions of small changes to wands damage).
- Icy Chill weapon enchant (has been used for some solo guides during wotlk/cata? era, doesn't mention frost damage specifically).

Please feel free to comment current state of CC PTR regarding non-confirmed issues. Would be nice to hear the opinion of CC staff what intention they want to follow (replicating old (bugged?) experience from 3.3.5; replicating Blizzard's claims nowadays of what was intended/bugged back then (classic?); replicating own interpretation of original 3.3.5 Blizzard's intention (making additional frost damage weapons (Iceblade Hacker, Warblade of Caer Darrow) work on Viscidus (might make this encounter too easy).

**Sources:**
- 3.0.9 claiming frost wands don't work: https://www.wowhead.com/wotlk/npc=15299/viscidus#comments:id=662215.
- Early classic frost wands don't work: https://www.icy-veins.com/forums/topic/51224-frost-wands-not-working-on-viscidus/ & https://www.wowhead.com/classic/news/classic-viscidus-possibly-bugged-frost-wands-not-counting-towards-freezing-the-317259 & https://www.reddit.com/r/classicwow/comments/i438ku/reports_coming_in_that_frost_wands_do_not_count/.
- Vanilla raid footage, nobody wanding, only frostbolts of mages being used, priest POV (could've used wand): https://www.youtube.com/watch?v=I0J9QDiZO34.
- Vanilla raid footage, wands being used (comments claiming that wands didn't contribute as the time to freeze him took longer as it should have: 0:30->1:30?) https://www.youtube.com/watch?v=XRRE4EM9yT0 & https://us.forums.blizzard.com/en/wow/en/wow/t/frost-wands-not-freezing-viscidus/603206/28 & https://us.forums.blizzard.com/en/wow/en/wow/t/frost-wands-not-freezing-viscidus/603206/62.
- Solo guide 2013 mentioning Icy Chill enchant, not mentioning frost wands: https://www.youtube.com/watch?v=tq2fXxq4yBQ.
- MadSeasonShow solo guides at level 100 not mentioning wands at all, e.g. warlock using melee weapon with spell damage enchant: https://youtube.com/playlist?list=PL0hAOfdoZQkK6FrdLllLqT1ISZf4SniA7.
- Blizzard claiming that frost wands not working on Viscidus is a bug and is fixed for classic (was it "bugged" during 3.3.5 era?) https://us.forums.blizzard.com/en/wow/t/frost-wands-not-freezing-viscidus/603206/87 (conflicting line of code from 2008).
- Blizzard saying additional frost damage on melee weapons not counting as frost damage "working as intended": https://us.forums.blizzard.com/en/wow/en/wow/t/weapons-with-frost-damage-or-any-other-school-dont-work/569190/7 (resulting in Blizzard removing the additional frost damage from those weapons tooltips alltogether for Classic).
- Retail 08/20 claiming frost wands don't work: https://us.forums.blizzard.com/en/wow/en/wow/t/frost-wands-not-freezing-viscidus/603206/21.
- Many more that are accessible by following the discussion of the threads above or simple research. Might post links later if additional information is found.

**Related issues:**
- https://github.com/chromiecraft/chromiecraft/issues/3610
- https://github.com/chromiecraft/chromiecraft/issues/4178
- https://github.com/chromiecraft/chromiecraft/issues/2298

Just wanted to share this with the initial thought of opening a bug report but am not sure after this research whom to believe so want to open this discussion to hear from others. Thank you for your work @ devs. Don't believe everything that Activision Blizzard says, they are not the 3.3.5 Blizzard.

**Comment by Cooties-cc** [comment]
It looks like the wands have been cleared up, and now do correct school damage. The only vague problem left is if Icy Chill did count towards Viscidus 200 frost attacks. It should also be cleared up with CC staff on where they draw the lines in what data they would accept. Since we could use data from actual vanilla/TBC (2004-2008 or so), Wotlk data (late 2008-2010ish), or vanilla/tbc/wotlk classic. Each version of the game might have people saying different things, and up until now, I haven't really seen any blizzard post about the subject to clarify. I would think that vanilla classic would have the best data tracking on whether or not it was working, as only videos or comments would be seen for older actual retail data, which could just end in hearsay. Maybe if it was an ElitistJerk post about the subject, would it hold some ground. 

I'm personally on the side of Icy Chill working on the boss, but it's mostly from monkey see monkey do on various videos and guides suggesting it works.
